GetandTake/Models/DTOs/BaseDTO/ProductDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using GetandTake.Models.Base;
+
+namespace GetandTake.Models.DTOs.BaseDTO
+{
+    public class ProductDTO : IBaseDTO
+    {
+        //use for Create,Update,Delete
+        public string ProductName { get; set; }
+        public string? QuantityPerUnit { get; set; }
+        public int? UnitPrice { get; set; }
```

Comment:
Well, just suggestion: for prices it's really better to use `decimal` type since prices can vary and be non-integer